src/App.css

```diff
@@ -16,6 +16,6 @@
   text-decoration: underline !important;
 }
 
-.active {
-  text-decoration: underline;
+.btn {
+  box-shadow: none !important;
```

Comment:
[Optional]
Try to avoid `!important` whenever necessary because it won't let anything else override it.

Reply:
I am using bootstrap so I have to override it. to get my desired look and feel.